onnxruntime/core/language_interop_ops/torch/refcount_tracker.cc

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <Python.h>
+#include <iostream>
+#include "core/language_interop_ops/torch/refcount_tracker.h"
+#include "core/platform/env.h"
+
+namespace onnxruntime {
+namespace language_interop_ops {
+namespace torch {
+#ifndef NDEBUG
+
+RefCountTracker::RefCountTracker() {
+  addr_info_map_ = {
+      {RefCountTracker::ObjCategory::ForwardArgs, forward_arg_addresses_},
+      {RefCountTracker::ObjCategory::ReturnValues, return_value_addresses_},
+      {RefCountTracker::ObjCategory::AutoGradContext, auto_grad_addresses_},
+  };
+}
+
+void RefCountTracker::TrackPyObject(RefCountTracker::ObjCategory category, PyObject* py_obj, std::string log_tag) {
+  AddressInfos& addrs = addr_info_map_[category];
+  void* addr = static_cast<void*>(py_obj);
```

Comment:
Assert py_obj != 0 (avoids the risk of the debug code below causing an error in Py_REFCNT)

Reply:
thanks!